### ISSUE TITLE
Split out into separate modules

### ISSUE THREAD

**Issue by stevenvachon** [posthtml/htmlnano#33]
For example, I have a need to merge all `<style>` elements for a different transformation before minifying the entire document. Currently, I'll need to run htmlnano with all sub-modules disabled except for `mergeStyles`, which poses a problem for a future where new sub-modules are added.

**Edit:**
`mergeStyles` could replace the lesser quality [posthtml-collect-styles](https://www.npmjs.com/package/posthtml-collect-styles).

**Comment by maltsev** [comment]
You can also use htmlnano modules separately:
```js
var posthtml = require('posthtml');
var mergeStyles = require('htmlnano/lib/modules/mergeStyles').default;
posthtml([mergeStyles]).process(html);
```

I also updated the README with that example.

**Comment by maltsev** [comment]
Though, we could consider adding an attribute which would disable all modules by default.

**Comment by maltsev** [comment]
This is going to be implemented in presets (#64).
